### PR TITLE
Update SECURE_KEY_PATTERN for Logger to hide <private-key> properties

### PR DESCRIPTION
**Pull request by tshkursk**

### CODE REVIEW

File: vividus/src/main/java/org/vividus/report/MetadataLogger.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
     private static final String NEW_LINE = "%n";
     private static final Logger LOGGER = LoggerFactory.getLogger(MetadataLogger.class);
     private static final Pattern SECURE_KEY_PATTERN = Pattern
-            .compile(".*(password|access-?key|api-?key|secret|token).*", Pattern.CASE_INSENSITIVE);
+            .compile(".*(password|access-?key|api-?key|secret|token|private-?key).*", Pattern.CASE_INSENSITIVE);
```
Comment (ikalinin1) [inline vividus/src/main/java/org/vividus/report/MetadataLogger.java:57]: what about `key` ?

Comment (tshkursk) [reply vividus/src/main/java/org/vividus/report/MetadataLogger.java:57]: I can add if really needed. Please, provide usage examples?

Comment (ikalinin1) [reply vividus/src/main/java/org/vividus/report/MetadataLogger.java:57]: The idea is to have a generic matcher for anything with the `key` part

Comment (tshkursk) [reply vividus/src/main/java/org/vividus/report/MetadataLogger.java:57]: The question is do you really want to hide everithing with `key` part?
Looks like lot of confings could be affected, e.g.: https://docs.vividus.dev/vividus/latest/plugins/plugin-azure-cosmos-db.html#_configuration

Comment (valfirst) [inline vividus/src/main/java/org/vividus/report/MetadataLogger.java:57]:
```suggestion
            .compile(".*(password|((access|api|private)-)?(key|secret|token)).*" Pattern.CASE_INSENSITIVE);
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**